features/ftue/impl/src/main/kotlin/io/element/android/features/ftue/impl/welcome/WelcomeView.kt

```diff
@@ -75,13 +75,15 @@ fun WelcomeView(
                 Text(
                     modifier = Modifier.testTag(TestTags.welcomeScreenTitle),
                     text = stringResource(R.string.screen_welcome_title, applicationName),
-                    style = ElementTheme.typography.fontHeadingLgBold,
+                    style = ElementTheme.typography.fontHeadingMdBold,
```

Comment:
Changing the font, see in [Figma](https://www.figma.com/file/0MMNu7cTOzLOlWb7ctTkv3/Element-X?node-id=8029%3A128756&mode=dev). (the iOS Figma was used I guess)